CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 # http-signature-header
 
+## 3.0.0 - TBD
+
+### Removed
+- **BREAKING**: `createAuthzHeader` has been removed.
+- **BREAKING**: `createSignatureString` has been removed.
+- **BREAKING**: CommonJS Module exports.
```

Comment:
Isn't this covered by the change to node >= 14? This is no longer a common js module. It might be possible to produce an older version using ESM and then export main via the `modules` so time permitting I will try some options to allow non-ES6 import/export statements today.